mozphab/detect_repository.py

```diff
@@ -17,7 +17,7 @@ def find_repo_root(path: str) -> Optional[str]:
     """Lightweight check for a repo in/under the specified path."""
     path = os.path.abspath(path)
     while os.path.split(path)[1]:
-        if Mercurial.is_repo(path) or Git.is_repo(path):
+        if Mercurial.is_repo(path) or Jujutsu.is_repo(path) or Git.is_repo(path):
```

Comment:
nitpick: I think, to be consistent with the other place where `Jujutsu` needs to come first, we should do the `Jujutsu` check first. I don't believe it matters in this case, but I think we might avoid potential surprises by keeping things consistent.